### DISCUSSION TITLE
Ability to run this package from outide vue components

### DISCUSSION THREAD

**Discussion by jet2018** [ankurk91/vue-toast-notification#87] [General]
Apparently, I have just realised that all methods are scoped on this.$toast, which is in the component scope. 
So am wondering if it is possible for me to call this package from my services(Typescript and Js classes) that are independent of vue components.

If it is already possible, please let me know how. Otherwise, we can see how we can achieve it.

**Comment by ankurk91** [comment]
If you are consuming this package in vue 3, then You can use composition api
https://github.com/ankurk91/vue-toast-notification#composition-api-usage
